### ISSUE TITLE
Build Error on Windows

### ISSUE THREAD

**Issue by geiseri** [itay-grudev/SingleApplication#53]
I see this error on windows builds:
```
singleapplication\singleapplication_p.cpp:104: error: use of undeclared identifier 'timeout'
```
It looks like it was removed from the API at some point, removing the Q_UNUSED lets it pass.

**Comment by geiseri** [comment]
This seems fixed in master, it just wasn't in 3.0.12a.

**Comment by itay-grudev** [comment]
I make it a case to digitally sign my work and at the moment of merging the PR I didn't have access to my key. I created a new unsigned release from master, which I'll sign when I have a chance.
